index.html

```diff
@@ -123,15 +135,17 @@ <h4>CANOPY</h4>
                     </div>
                     <ul class="small-btn-div">
                         <li class="small-btn"><a>html</a></li>
+                        <li class="small-btn small-btn-dis"><a>Ruby on rails</a></li>
                         <li class="small-btn"><a>css</a></li>
                         <li class="small-btn"><a>javaScript</a></li>
                     </ul>
                     <button><a>See Project</a></button>
                 </div>
             </div>
-            <div class="grid-item grid-item-4">
+            <div  id="reverse" class="grid-item grid-item-4">
                 <div class="card-box">
-                    <img src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                    <img class="mobile-image" src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                    <img class="desktop-image" src="images/Snapshoot-Portfolio5.png" alt="" srcset="">
                 </div>
                 <div class="text-box">
                     <h2>Multi-Post<br>Stories</h2>
```

Comment:
- [ ] Kindly follow the design template when designing the header of the cards with the title **_"Multi-Post Stories"_**

### Your Design

![image](https://user-images.githubusercontent.com/77106962/214149442-5d0004ed-4dd7-433d-8470-1cbceab3774f.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/214149558-e83414eb-1244-492b-a541-064590eb5f8d.png)